src/ledger/LedgerTxn.cpp

```diff
@@ -2672,7 +2711,24 @@ LedgerTxnRoot::Impl::putInEntryCache(
 {
     try
     {
-        mEntryCache.put(key, {entry, type});
+        mEntryCache.put(key, CacheEntry{entry, type});
```

Comment:
I think those metrics got removed in a previous commit, and I am not sure I'd want to pay the price for them in the general case (this cache is very performance sensitive).

So either find a way to only compute those things on demand, or don't compute them at all.